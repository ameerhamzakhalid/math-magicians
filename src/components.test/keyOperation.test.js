import React from 'react';
import { render } from '@testing-library/react';
import KeyOperation from '../components/keyOperation';

describe('KeyOperation', () => {
  it('matches snapshot', () => {
    const clickEvent = jest.fn();
    const { container } = render(<KeyOperation clickEvent={clickEvent} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
