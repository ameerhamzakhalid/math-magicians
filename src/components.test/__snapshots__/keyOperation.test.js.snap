// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KeyOperation matches snapshot 1`] = `
<div
  class="container-buttons"
>
  <button
    class="button"
    id="ac"
    type="button"
  >
    AC
  </button>
  <button
    class="button"
    id="plusAndMinus"
    type="button"
  >
    +/-
  </button>
  <button
    class="button"
    id="percentage"
    type="button"
  >
    %
  </button>
  <button
    class="button"
    id="divide"
    type="button"
  >
    รท
  </button>
  <button
    class="button"
    id="seven"
    type="button"
  >
    7
  </button>
  <button
    class="button"
    id="eight"
    type="button"
  >
    8
  </button>
  <button
    class="button"
    id="nine"
    type="button"
  >
    9
  </button>
  <button
    class="button"
    id="multiply"
    type="button"
  >
    x
  </button>
  <button
    class="button"
    id="four"
    type="button"
  >
    4
  </button>
  <button
    class="button"
    id="five"
    type="button"
  >
    5
  </button>
  <button
    class="button"
    id="six"
    type="button"
  >
    6
  </button>
  <button
    class="button"
    id="minus"
    type="button"
  >
    -
  </button>
  <button
    class="button"
    id="one"
    type="button"
  >
    1
  </button>
  <button
    class="button"
    id="two"
    type="button"
  >
    2
  </button>
  <button
    class="button"
    id="three"
    type="button"
  >
    3
  </button>
  <button
    class="button"
    id="plus"
    type="button"
  >
    +
  </button>
  <button
    class="button"
    id="zero"
    type="button"
  >
    0
  </button>
  <button
    class="button"
    id="full-stop"
    type="button"
  >
    .
  </button>
  <button
    class="button"
    id="equal-to"
    type="button"
  >
    =
  </button>
</div>
`;
